sources/platform/actors/security.md

```diff
@@ -11,6 +11,24 @@ slug: /actors/security
 
 ---
 
+## Security Whitepaper
+
 At Apify, security is the top priority of our daily work. Security best practices are reflected in our development, deployment, monitoring, and project management processes.
+Read the Apify Security Whitepaper for a full description of Apify's measures and commitments to security:
+
+<a href="https://apify.com/security-whitepaper.pdf" target="_blank" title="Apify Security Whitepaper">
+    <img src={require("./images/security/whitepaper-cover.png").default} width="50%" title="Apify Security Whitepaper" />
+</a>
+
+## Reporting a vulnerability
+
+If you notice or suspect a potential security breach, please report this immediately to our security team at [security@apify.com](mailto:security@apify.com), including all the technical details.
+
+The total compensation for your report will be highly dependent on the severity, complexity, and exploitability of the vulnerability. Your reports are always verified by our internal security team,
```

Comment:
The total compensation for your report will be highly dependent on the vulnerability's severity, complexity, and exploitability. Our internal security team always verifies your reports.